### PR TITLE
PUB-969 - Updates to search a court page

### PR DESCRIPTION
**Pull request by junaidiqbalmoj**
Image 22-12-2021 at 11 55

Change description
Remove the Search option page and move A-Z courts link page to court search page.

- [x] commit messages are meaningful and follow good commit message guidelines
- [x ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

https://tools.hmcts.net/jira/browse/PUB-969

**Does this PR introduce a breaking change?** (check one with "x")

[ ] Yes
[x] No

### CODE REVIEW

File: src/test/e2e/pageobjects/Search.page.ts [hunk 1]
```diff
@@ -31,4 +32,15 @@ export class SearchPage extends CommonPage {
     button.click();
     return new HearingListPage();
   }
+  SearchAToZLink
```
Comment (ChrisS1512) [inline src/test/e2e/pageobjects/Search.page.ts:35]: Line 35 can be removed I think :)

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

File: src/test/e2e/pageobjects/Search.page.ts (hunk 1 above)
Comment (junaidiqbalmoj) [reply src/test/e2e/pageobjects/Search.page.ts]: fixed :)

**Review by ChrisS1512 — APPROVED** [review]
Looks good :)

**Review by furnivall — APPROVED** [review]
lgtm

**Pull request merged**